presto-product-tests/bin/run_on_docker.sh

```diff
@@ -159,6 +168,10 @@ docker version
 
 stop_all_containers
 
+if [[ "$CONTINUOUS_INTEGRATION" == 'true' ]]; then
```

Comment:
Unfortunately, jenkins is a different beast. Below is the list of env vars of an empty build in default jenkins install. So I'd stick with the first version. Or expand it for jenkins, if you guys are running docker on jenkis :)

```
+ env
JENKINS_HOME=/var/jenkins_home
HOSTNAME=d1e123875eb0
NODE_LABELS=master
SHLVL=0
HOME=/var/jenkins_home
HUDSON_COOKIE=79b2dfe4-a3c8-4dd3-bce0-77063be5d178
JENKINS_SERVER_COOKIE=a4e5ef96de976ee2
JENKINS_UC=https://updates.jenkins.io
WORKSPACE=/var/jenkins_home/workspace/asdf
JAVA_VERSION=8u102
NODE_NAME=master
CA_CERTIFICATES_JAVA_VERSION=20140324
EXECUTOR_NUMBER=0
TINI_SHA=fa23d1e20732501c3bb8eeeca423c89ac80ed452
JENKINS_VERSION=2.7.2
BUILD_DISPLAY_NAME=#2
HUDSON_HOME=/var/jenkins_home
JOB_BASE_NAME=asdf
JAVA_DEBIAN_VERSION=8u102-b14.1-1~bpo8+1
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
BUILD_ID=2
BUILD_TAG=jenkins-asdf-2
LANG=C.UTF-8
BUILD_NUMBER=2
JENKINS_SLAVE_AGENT_PORT=50000
HUDSON_SERVER_COOKIE=a4e5ef96de976ee2
JOB_NAME=asdf
COPY_REFERENCE_FILE_LOG=/var/jenkins_home/copy_reference_file.log
JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
PWD=/var/jenkins_home/workspace/asdf
TINI_VERSION=0.9.0
Finished: SUCCESS
```